deploy/Dockerfile

```diff
@@ -1,9 +1,17 @@
 FROM quay.io/openshift/origin-operator-registry:latest
 
-COPY deploy/olm-catalog /registry
+WORKDIR /registry
+
+COPY deploy/olm-catalog .
```

Comment:
Isn't it the same result? Why did you split `COPY deploy/olm-catalog /registry` into two dockerfile commands?

Reply:
Yes and no... yes because the end result will be the same - having /registry/kubevirt-hyperconverged, and no because we need WORKDIR to be set for the RUN and ENTRYPOINT (it was set previously on the operator-registry as well), but it was removed.  The alternative would be to keep the COPY command as it is, and move WORKDIR after, does it matter ?

Reply:
/lgtm